static/app/components/searchSyntax/grammar.pegjs

```diff
@@ -217,11 +217,24 @@ aggregate_key
     }
 
 function_args
-  = arg1:key
-    args:(spaces comma spaces key)* {
+  = arg1:aggregate_param
+    args:(spaces comma spaces aggregate_param)* {
       return tc.tokenKeyAggregateArgs(arg1, args);
     }
 
+aggregate_param
+  = quoted_aggregate_param / raw_aggregate_param
+
+raw_aggregate_param
+  = param:[^()\t\n, \"]+ {
+    return tc.tokenKeySimple(param.join(''), false);
```

Comment:
Don't have prettier here, but this actually needs two more spaces in front of it. Sorry for the  nit

Reply:
nah, nit away, appreciate it 🙏

though 🤔 is there a way to get prettier in here so I don't have to think about styling? 😁

Reply:
There is a prettier plugin.. but it’s kinda.. questionable